### ISSUE TITLE
Filtering duplicate feed update news based on url

### ISSUE THREAD

**Issue by bw4518** [QuiteRSS/quiterss#1509]
In case I haven't overlooked this, I'd welcome an option to not add the news again, if only the title/description/etc changes, but the url is the same.

**Comment by ckasprzak** [comment]
Yeah I would like this as a new feature! Time consuming going through two or three copies of the same feed article!

**Comment by simevo** [comment]
I too find this feature very useful. Thanks for any suggestion how to address it!

**Comment by cuixiping** [comment]
This bug is still here in 2026.
Also see #874
